### PR TITLE
Dockerfile

### PR DESCRIPTION
**Pull request by hannah98**
I have created a lightweight Dockerfile for the python app.  It is ~140MB.  It could be smaller if this project used a requirements.txt file because then pipenv would not need to be installed.  However, I see you just accepted a pull request to retire the requirements.txt for a pipfile so I just went with installing pipenv.

A nice advantage of using Docker is that you can run multiple instances at the same time by just varying the public port number - the internal port number will always be 5000.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,29 @@ $ pipenv install
     * `$ pipenv run python blockchain.py` 
     * `$ pipenv run python blockchain.py -p 5001`
     * `$ pipenv run python blockchain.py --port 5002`
+    
+## Docker
+
```
Comment (dvf) [inline README.md]: Let's add the following to show that running it in Docker is entirely optional:

```
If you'd like to use Docker, you can follow these instructions... etc.
```

**Comment by dvf** [comment]
Thanks for this! Had some comments

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+FROM python:3-alpine
```
Comment (dvf) [inline Dockerfile]: Let's use `python:3.6-alpine`.

Comment (hannah98) [reply Dockerfile]: Agreed - I have updated the Dockerfile

File: README.md (hunk 1 above)
Comment (hannah98) [reply README.md]: Good point - I have updated the readme to reflect this.

**Pull request merged**